AdminWebsite/AdminWebsite/Mappers/AddNotificationRequestMapper.cs

```diff
@@ -316,21 +316,24 @@ public static class AddNotificationRequestMapper
             {
                 notificationType = NotificationType.StaffMemberDemoOrTest;
                 parameters.Add(NotifyParams.StaffMember, $"{participant.FirstName} {participant.LastName}");
-            }
+            }            
             else if (participant.UserRoleName.Contains(RoleNames.Judge, StringComparison.InvariantCultureIgnoreCase))
             {
-                if (hearing.IsJudgeEmailEJud())
-                {
-                    notificationType = NotificationType.EJudJudgeDemoOrTest;
-                }
-                else
-                {
-                    notificationType = NotificationType.JudgeDemoOrTest;
-                    parameters.Add(NotifyParams.CourtroomAccountUserName, participant.Username);
-                }
+                var judgeEmail = hearing.GetJudgeEmail();
+                if (string.IsNullOrEmpty(judgeEmail)) return null;
```

Comment:
```suggestion
                if (string.IsNullOrEmpty(judgeEmail)) 
                {
                    return null;
                }
```

The above is a personal preference of mine maintainability. But we don't have any style guidelines regarding this, so you can leave it if you want. But the following lines should be unindented as at first glance it looks like it's nested inside this.